Containerfile.rag

```diff
@@ -11,12 +11,15 @@ USER 0
 RUN microdnf install -y unzip wget \
   && mkdir -p vector-db \
 	&& cd vector-db \
-  && wget $RAG_INDEX \
-	&& unzip local.zip \
+  && wget -q $RAG_INDEX \
+	&& unzip -qq local.zip \
 	&& mv local ocp-product-docs \
 	&& rm -f local.zip \
 	&& microdnf remove -y unzip wget
 
+COPY scripts/download_embeddings_model.py .
+RUN python download_embeddings_model.py embeddings_model && rm download_embeddings_model.py
```

Comment:
i had some debate w/ myself about whether this belongs in the base containerfile or this one.  but it's ok for now.  Eventually we'll probably be back to just having one containerfile, and it will do all the steps (pull in deps, pull in models, pull in rag content).